src/js/common/schemaform/helpers.js

```diff
@@ -539,3 +539,41 @@ export function setItemTouched(prefix, index, idSchema) {
     return _.merge(idObj, setItemTouched(prefix, index, idSchema[field]));
   }, {});
 }
+
+export function replaceRefSchemas(schema, definitions, path = '') {
+  if (schema.$ref) {
```

Comment:
If we start building schemas from classes and/or with prototypes, I will complain loudly. :-)